legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-grammar/src/main/antlr4/org/finos/legend/engine/language/pure/grammar/from/antlr4/RelationalParserGrammar.g4

```diff
@@ -44,6 +44,11 @@ database:                                   DATABASE qualifiedName
 include:                                    INCLUDE qualifiedName
 ;
 
+// -------------------------------------- STEREOTYPE --------------------------------------
+
+stereotypes:                        LESS_THAN LESS_THAN stereotype (COMMA stereotype)* GREATER_THAN GREATER_THAN;
+
+stereotype:                         qualifiedName DOT identifier;
```

Comment:
Rather than duplicating this, we should reuse from M3Grammar/DomainGrammar

Reply:
Adding that import might introduce complications due to potential keyword conflicts and ambiguous references as DomainParserGrammar -> M3ParserGrammar -> CoreParserGrammar and RelationalParserGrammar -> CoreParserGrammar